packages/replay-next/components/lexical/CommentEditor.tsx

```diff
@@ -155,18 +155,23 @@ export default function CommentEditor({
 
   // Toggle editor's enabled state when the prop changes.
   useEffect(() => {
+    let timeOutID: any = null;
     const editor = editorRef.current;
     if (editor != null) {
       if (editor.isEditable() !== editable) {
         editor.setEditable(editable);
 
         if (editable) {
-          setTimeout(() => {
+          timeOutID = setTimeout(() => {
             editor.focus();
           });
         }
```

Comment:
I think the `timeoutId` variable and effect cleanup function could be moved inside of this conditional. The effect only needs to return that cleanup if it actually creates the timeout.